.gitignore

```diff
@@ -147,6 +147,13 @@ examples/hiop/ex9-mesh.*
 examples/hiop/ex9-init.*
 examples/hiop/ex9-final.*
 
+examples/ex71
+examples/pLap/
+examples/ex71p
+examples/pLap/*
+examples/pLap
+examples/pLap/
```

Comment:
File naming is not in convention with other examples

Reply:
The correct number will be set just before merging as we do not know yet the last number.

Reply:
I was referring to the pLap directory which is created

Reply:
Everything should be OK now.